boards/arm/xenvm/doc/index.rst

```diff
@@ -0,0 +1,179 @@
+
++--------------+-------------+----------------------+
+| Interface    | Controller  | Driver/Component     |
++==============+=============+======================+
+| GIC          | virtualized | interrupt controller |
++--------------+-------------+----------------------+
+| SBSA UART    | emulated    | serial port          |
++--------------+-------------+----------------------+
+| ARM TIMER    | virtualized | system clock         |
++--------------+-------------+----------------------+
+
+The kernel currently does not support other hardware features on this platform.
+
+Devices
+========
+System Clock
+------------
+
+This board configuration uses a system clock frequency of 8.32 MHz. This is the
+default value, which should be corrected for user's actual hardware.
+
+You can determine clock frequency of your ARM Generic Timer by inspecting Xen
+boot log:
+
+::
+
+  (XEN) [    0.147541] Generic Timer IRQ: phys=30 hyp=26 virt=27 Freq: 8320 KHz
+
+Serial Port
+-----------
+
+This board configuration uses a single serial communication channel using SBSA
+UART. This is a minimal UART implementation provided by Xen. Xen PV Console is
+not supported at this moment.
+
+Interrupt Controller
+--------------------
+
+By default, GICv2 is selected. If your hardware is based on GICv3, you can
+configure Zephyr to use it, by amending device tree and Kconfig
+option in "xenvm" SoC as well as guest configuration file.
+
+CPU Core type
+-------------
+
+Default core in this configuration is Cortex A72. Depending on yours actual
+hardware you might want to change this option in the same way as Interrupt
+Controller configuration.
+
+Known Problems or Limitations
+==============================
+
+Xen configures guests in runtime by providing device tree that describes guest
+environment. On other hand, Zephyr uses static configuration that should be know
+at build time. So there are chances, that Zephyr image created with default
+configuration would not boot on your hardware. In this case you need to update
+configuration by altering device tree and Kconfig options. This will be covered
+in detail in next section.
+
+No Xen-specific features are supported at the moment. This includes:
+
+* Xen Enlighten memory page
+* XenBus
+* Xen event channels
+* Xen grant tables
+* Xen PV drivers (including PV console)
+
+Building and Running
+********************
+
+Use this configuration to run basic Zephyr applications and kernel tests as Xen
+guest, for example, with the :ref:`synchronization_sample`:
+
+
+.. code-block::
+
+   $ west build -b xenvm samples/synchronization
+
+This will build an image with the synchronization sample app. Next, you need to
+create guest configuration file :code:`zephyr.conf`. There is example:
+
+.. code-block::
+
+   kernel="zephyr.bin"
+   name="zephyr"
+   vcpus=1
+   memory=16
+   gic_version="v2"
+   on_crash="preserve"
+   vuart="sbsa_uart"
+
+You need to upload both :code:`zephyr.bin` and :code:`zephyr.conf` to your Dom0
```

Comment:
I don't see any `zephyr.bin` as output of `west ..`, only `zephyr.elf`. Any missing steps here?